lib/driver.js

```diff
@@ -1118,8 +1118,22 @@ class XCUITestDriver extends BaseDriver {
 
     if (this.isRealDevice()) {
       await installToRealDevice(this.opts.device, this.opts.app, this.opts.bundleId, this.opts.noReset);
+      if (this.opts.otherApps) {
+        log.warn('Capability otherApps is only supported for Simulators');
+      }
     } else {
       await installToSimulator(this.opts.device, this.opts.app, this.opts.bundleId, this.opts.noReset);
+      if (this.opts.otherApps) {
+        let otherApps;
+        try {
+          otherApps = parseArray(this.opts.otherApps);
+        } catch (e) {
+          log.errorAndThrow(`Could not parse "otherApps" capability: ${e.message}`);
+        }
+        for (let otherApp of otherApps) {
+          await installToSimulator(this.opts.device, otherApp, undefined, this.opts.noReset);
```

Comment:
is there a reason we need to pass `undefined` instead of `null`? i don't like using `undefined` proactively as a value.

Reply:
I can change it to `null`, wanted to throw this PR for quick feedback and check what you guys think.